concepts/05 Widgets/Scheduler/030 Appointments/015 Appointment Types/020 All-Day Appointments.md

```diff
@@ -1,25 +1,27 @@
-An all-day appointment is a non-recurring appointment that covers the whole [timetable's period](/concepts/05%20Widgets/Scheduler/050%20Timetable.md '/Documentation/Guide/Widgets/Scheduler/Timetable/').
+An all-day appointment is an appointment that occurs during the whole day.
 
 ![Scheduler All-Day Appointment](/images/UiWidgets/Scheduler_AllDayAppointment.png)
 
-An appointment can also be marked as all-day by assigning **true** to the defining object's [allDay](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/allDay.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay') field. The [startDate](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/startDate.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDate') field should also be present in this object; the [endDate](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/endDate.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#endDate') is optional.
+To create such an appointment, set its [allDay](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#allDay) field to **true** and specify the start day in the [startDate](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDate) field; the time value is optional: 
 
     <!--JavaScript-->
     var allDayAppointment = [{
-        text: "Fix bugs",
-        startDate: new Date(2016, 4, 10),
+        text: "Concert",
+        startDate: new Date("2021-07-27T16:00:00.000Z"),
         allDay: true
     }];
 
-If appointments in your data source omit the **allDay** field but have another field instead, assign its name to the [allDayExpr](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/allDayExpr.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#allDayExpr') option.
+[note]If you set the optional [endDate](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#endDate) field to a date after the start date, the appointment will last for more than one whole days.
+
+If your appointment data objects already have a field that acts as **allDay**, specify it in the [allDayExpr](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#allDayExpr) option of the **Scheduler**:
```

Comment:
I would rewrite the first part of the sentence. I'm not sure how exactly, but it is not really clear how a field can act as **allDay**. Maybe something like this:

If appointment data objects have a field that indicate whether the appointment lasts the whole day, specify this field in...